qiskit/providers/models/pulsedefaults.py

```diff
@@ -94,44 +98,335 @@ class Command(BaseModel):
+        """
+        Is the operation defined for the given qubits?
+
+        Args:
+            operation: The operation for which to look.
+            qubits: The specific qubits for the operation.
 
         Returns:
-            CmdDef: `CmdDef` instance generated from defaults
+            True iff the operation is defined.
+        """
+        return operation in self._ops_def and \
+            _to_tuple(qubits) in self._ops_def[operation]
+
+    def assert_has(self, operation: str, qubits: Union[int, Iterable[int]]) -> None:
+        """
+        Convenience method to check that the given operation is defined, and error if it is not.
+
+        Args:
+            operation: The operation for which to look.
+            qubits: The specific qubits for the operation.
+
+        Returns:
+            None
+
+        Raises:
+            PulseError: If the operation is not defined on the qubits.
+        """
+        if not self.has(operation, _to_tuple(qubits)):
+            if operation in self._ops_def:
+                raise PulseError("Operation '{op}' exists, but is only defined for qubits "
+                                 "{qubits}.".format(op=operation,
+                                                    qubits=self.qubits_with_op(operation)))
+            raise PulseError("Operation '{op}' is not defined for this "
+                             "system.".format(op=operation))
+
+    def get(self,
+            operation: str,
+            qubits: Union[int, Iterable[int]],
+            *params: List[Union[int, float, complex]],
+            **kwparams: Dict[str, Union[int, float, complex]]) -> Schedule:
         """
-        return CmdDef.from_defaults(self.cmd_def, self.pulse_library)
+        Return the defined Schedule for the given operation on the given qubits.
+
+        Args:
+            operation: Name of the operation.
+            qubits: The qubits for the operation.
+            *params: Command parameters for generating the output schedule.
+            **kwparams: Keyworded command parameters for generating the schedule.
+
+        Returns:
+            The Schedule defined for the input.
+
+        Raises:
+            PulseError: If the operation is not defined on the qubits.
+        """
+        self.assert_has(operation, qubits)
+        schedule = self._ops_def[operation].get(_to_tuple(qubits))
+        if isinstance(schedule, ParameterizedSchedule):
+            schedule = schedule.bind_parameters(*params, **kwparams)
+        return schedule
+
+    def get_parameters(self, operation: str, qubits: Union[int, Iterable[int]]) -> Tuple[str]:
+        """
+        Return the list of parameters taken by the given operation on the given qubits.
+
+        Args:
+            operation: Name of the operation.
+            qubits: The qubits for the operation.
+
+        Returns:
+            The names of the parameters required by the operation.
+
+        Raises:
```

Comment:
I'm not sure this is appropriate to document the raises since it is not explicitly raising in the method, the linter might complain.

Reply:
Good catch, I think I wrote this before I wrote `assert_has`. I'll check that this isn't an issue in other methods too